pathd/path_pcep_debug.c

```diff
@@ -165,6 +166,7 @@ const char *pcep_event_type_name(pcep_event_type event_type)
 const char *pcep_error_type_name(enum pcep_error_type error_type)
```

Comment:
This function is no longer needed as it is [implemented in the PCEPlib](https://gitlab.voltanet.io/volta-controlplane/pceplib/-/merge_requests/70), where it actually should be.

Reply:
As far as I can see, the function in pceplib gives a description of the error and this function gives just a small identifier of the error. This is not very practical in the context of the debug information as it can get large. Am I missing something ?